docs/Lua API.rst

```diff
@@ -4415,12 +4415,29 @@ General script API
   command-line. The ``name`` argument should be the name of the script without
   its extension, as would be used on the command line.
 
-  Note that this function lets Lua errors propagate to the caller.
+  Example::
+     dfhack.run_script("armoks-blessing", "all")
+  Scipts that use a dash(``-``) in their arguments still require one when called
+  from ``dfhack.run_script``.
```

Comment:
This is somewhat more specific than it needs to be - something like "arguments must be entered in the same way as in the DFHack console" could work, but I'm not sure we even need this at all, because there isn't a way to run commands that does change arguments before they get passed to commands.

The only meaningful difference I can think of between `run_script` and the console is that you can pass in spaces within an argument to `run_script` without needing to escape them, but I'm not sure of a script that would benefit from this off the top of my head.